src/components/GamesList/GamesList.css

```diff
@@ -0,0 +1,7 @@
+.ui.black.inverted.right.aligned.segment {
+    transform: none;
+}
+
+.game-list-row {
+    margin: .75rem;
+}
```

Comment:
Are we sure we need this? Usually these frameworks have a mechanism for doing stuff like this. We should prefer the frameworks way of doing things whenever we can.

That said, if this is the way we have to do this then that's fine.